kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/security/CertificateListTabUi.java

```diff
@@ -237,70 +238,73 @@ public String getValue(GwtKeystoreEntry object) {
         this.certificatesGrid.setSelectionModel(this.selectionModel);
     }
 
-    private <U extends Comparable<U>> Comparator<GwtKeystoreEntry> getComparator(Function<GwtKeystoreEntry, U> comparableElementSupplier) {
-    	return new Comparator<GwtKeystoreEntry>() {
-
-			@Override
-			public int compare(GwtKeystoreEntry o1, GwtKeystoreEntry o2) {
-				if(o1 == o2)
-					return 0;
-				if(o1 == null)
-					return -1;
-				if(o2 == null)
-					return 1;
-				
-				U item1 = comparableElementSupplier.apply(o1);
-				U item2 = comparableElementSupplier.apply(o2);
-				
-				if(item1 == item2)
-					return 0;
-				if(item1 == null)
-					return -1;
-				if(item2 == null)
-					return 1;
-				
-				return item1.compareTo(item2);
-			}
-    	};
+    private <U extends Comparable<U>> Comparator<GwtKeystoreEntry> getComparator(
+            Function<GwtKeystoreEntry, U> comparableElementSupplier) {
+        return (o1, o2) -> {
```

Comment:
Formatting seems strange

Reply:
I see it properly in my eclipse

Reply:
i have checked formatting and indentention and i tseems ok to me, re-applying the formatter doesn't change anything in the file